recipe/conda_build_config.yaml

```diff
@@ -9,3 +9,11 @@ c_compiler:                # [win]
   - vs2022                 # [win]
 cxx_compiler:              # [win]
   - vs2022                 # [win]
+cudnn:                                            # [linux64]
+  - undefined                                     # [linux64]
+cuda_compiler_version:                            # [linux64]
+  - None                                          # [linux64]
+docker_image:                                     # [linux64]
+  - quay.io/condaforge/linux-anvil-cos7-x86_64    # [linux64]
+cdt_name:                                         # [linux64]
+  - cos7                                          # [linux64]
```

Comment:
Appears the bot is adding this. Following up in upstream issue ( https://github.com/regro/cf-scripts/issues/1981 )

Though don't think we need this logic. Would propose dropping and re-rendering

```suggestion
```

Reply:
This behavior changed in the bot with PR ( https://github.com/regro/cf-scripts/pull/1982 )

Reply:
Confirmed as fixed